### PR TITLE
fix(apim): minor fix APIM readme and help document

### PR DESCRIPTION
**Pull request by dooriya**

### CODE REVIEW

File: docs/fx-core/apim-help.md [hunk 1]
```diff
@@ -18,26 +18,21 @@ The Azure Active Directory application with object id '{objectId}' could not be
...
-Please register the resource provider namespace 'Microsoft.ApiManagement' for your subscription according to this [document](https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/error-register-resource-provider#solution-3---azure-portal).
 
-### Error Message
-Failed to import API Management API. [Detail] One or more fields contain incorrect values. {reason}
-### Mitigation
-The OpenAPI document is invalid. Please change the OpenAPI document according to the reason in the error message. The OpenAPI limitation in Azure API Management can be found [here](https://docs.microsoft.com/en-us/azure/api-management/api-management-api-import-restrictions).
+| Error Message | Mitigation |
+| :-------------| :----------|
+|Failed to create API Management Service. The subscription is not registered to use namespace 'Microsoft.ApiManagement'. See https://aka.ms/rps-not-found for how to register subscriptions.| Please register the resource provider namespace 'Microsoft.ApiManagement' for your subscription according to this [document](https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/error-register-resource-provider#solution-3---azure-portal)|
```
Comment (kimizhu) [inline docs/fx-core/apim-help.md:24]: to this [document].

Comment (kimizhu) [inline docs/fx-core/apim-help.md:24]: Remove "please"?

Comment (dooriya) [reply docs/fx-core/apim-help.md:24]: Fixed: add period after document.

Comment (dooriya) [reply docs/fx-core/apim-help.md]: Done.

**Pull request merged**